showkase/src/main/java/com/airbnb/android/showkase/ui/ShowkaseBrowserApp.kt

```diff
@@ -91,8 +92,7 @@ internal fun ShowkaseAppBar(
     Row(
         Modifier.fillMaxWidth()
             .graphicsLayer(shadowElevation = 4f)
-            .padding(padding2x)
-            .height(64.dp),
+            .padding(padding2x),
```

Comment:
we might want to add a maxSize for the height but I'm curious to see the screenshots before that

Reply:
hmm yeah maybe you're right 🤔 Is it also maybe a bit weird that it spans 3 lines instead of 2? What do you think? :)